Makefile

```diff
@@ -1,6 +1,6 @@
 AR ?= ar
 CC ?= gcc
-CFLAGS = -Ideps -lm -lpthread -pedantic -std=c99 -v -Wall -Wextra
```

Comment:
@nderjung Why did you remove the `-v` flag (for verbosity)?

Reply:
My mistake; I removed it because I didn't need it to be overly verbose.